prac_07/guitar.py

```diff
@@ -0,0 +1,33 @@
+"""Guitar class for cp1404 prac_07"""
+
+CURRENT_YEAR = 2023
+
+
+class Guitar:
+    """Represent a guitar object"""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a guitar instance"""
+        self.name = name
+        self.year = int(year)
```

Comment:
Don't think you need to say int(year) as up in the parameters year=0 means it initialised to an integer 😄